tests/e2e/archetype-plugin-e2e-tests.spec.ts

```diff
@@ -32,14 +33,31 @@ describe('Archetype Plugin E2E Testing for Taqueria CLI', () => {
 		await cleanup();
 	});
 
+	test('archetype plugin will support cli options', async () => {
+		const { execute, cleanup, spawn } = await prepareEnvironment();
+		const { waitForText } = await spawn('taq', 'init test-project --debug');
+		await waitForText("Project taq'ified!");
+
+		const { stdout } = await execute('taq', 'install ../taqueria-plugin-archetype', './test-project');
+		expect(stdout).toContain('Plugin installed successfully');
+
+		const { stdout: stdout2 } = await execute('taq', '--version', './test-project');
+		expect(stdout2).not.toEqual([]);
```

Comment:
Should we also assert that nothing was output to stderr?

Reply:
i can, but it implied